radio/src/telemetry/frsky.cpp

```diff
@@ -629,6 +629,14 @@ void telemetryInit(uint8_t protocol)
   else {
     telemetryPortInit(FRSKY_SPORT_BAUDRATE);
   }
+
+#if defined(REVX)
+  if (g_model.telemetryProtocol == PROTOCOL_FRSKY_D_INVERTED) {
+    setMFP();
+  } else {
```

Comment:
Noted. Will be a hard one, as it is different from how it is done at my work. If I start using eclipse, auto format will be very handy.